### DISCUSSION TITLE
Unable cofigure servcies

### DISCUSSION THREAD

**Discussion by maglini** [OrchardCMS/OrchardCore#9875] [Q&A]
Startup:
 services
     .AddSetup()
     .AddOrchardCore()
     .AddMvc()
     .WithTenants();

![image](https://user-images.githubusercontent.com/60664635/124778791-e99e3b80-df49-11eb-83f7-15faa6edf11f.png)

**Comment by ns8482e** [comment]
If you are creating MVC application for OrchardCore Framework use command line tool for code generation templates. It's quick and error free to start :)

```bash
dotnet new ocmvc
```
[Documented here](https://docs.orchardcore.net/en/dev/docs/getting-started/templates/index.html#code-generation-templates)
